srcmedia/ts/members-search.ts

```diff
@@ -21,7 +21,9 @@ document.addEventListener('DOMContentLoaded', () => {
     const $sortSelect = document.querySelector('select[name=sort]') as HTMLSelectElement
     const $pageControls = document.getElementsByClassName('sort-pages')[0] as HTMLElement
     const $genderFacet = document.querySelector('#id_sex') as HTMLFieldSetElement
-
+    // FIXME should use an id selector
```

Comment:
it definitely will be, yep. this is the reason I was asking if the widget could put an id on the `<fieldset>` like `<fieldset id="id_gender">` or similar. did we decide that it shouldn't be prefixed with `id_`?

Reply:
@thatbudakguy oh, we discussed but it never got added.  Seems good to keep the id prefix for consistency; does it seem fine to generate as `"id_" + field.name` ?

It also occurs to me that the gender filter widget _includes_ the fieldset and legend, but the date range does not. I think this is fine for now (and I _think_ not including is more consistent with standard django widgets), but we may want to revisit as we add other range filters.

Reply:
cool, I'll add the id. one other somewhat related thing...how would you feel about (eventually) wrapping the "has card" facet in a `<fieldset>`? my arguments for it are:
- it's technically a facet since it has a count, therefore it corresponds to a `<fieldset>`
- the count is technically an `<output>` (i should change them so they're not just `<span>`), which means the `<fieldset>` is doing its job by wrapping an `<input>` and its related `<output>`, not just a single `<input>`
- that way we don't have an `<input>` and its associated `<label>` just sort of floating out there without an element wrapping/relating them, since they're positioned next to each other to take advantage of a styling trick with the `:checked` selector

Reply:
Using fieldsets sounds good to me. I wonder if we could do it as a common template snippet rather than including it in the widget proper?

Reply:
makes more sense to me to actually include it in the widget proper, since it will have classes applied (`<fieldset class="range facet">`) and a `<legend>` specific to the stuff inside.

Reply:
moved this discussion to #326